build.ps1

```diff
@@ -3,7 +3,7 @@ cd $PSScriptRoot
 $repoFolder = $PSScriptRoot
 $env:REPO_FOLDER = $repoFolder
 
-$koreBuildZip="https://github.com/aspnet/KoreBuild/archive/1.0.0.zip"
+$koreBuildZip="https://github.com/aspnet/KoreBuild/archive/feature/msbuild.zip"
```

Comment:
FYI, this version is obsolete and will be removed in the future. You need the `dev` version: https://github.com/aspnet-contrib/AspNet.Security.OAuth.Extensions/pull/63/files#diff-f0e02e42e53ec40f5d3b8bed031ad09e